projects/static/projects/js/members.js

```diff
@@ -0,0 +1,28 @@
+function resetRole(selector_id) {
+  var selector = document.getElementById(selector_id);
+  for (var j = 0; j < selector.options.length; j++) {
+    var opt = selector.options[j];
+    if (opt.disabled) {
+          selector.value = opt.value;
+          break;
+    }
+  }
+  selector.disabled = true;
+  document.getElementById("submit_" + selector.id).style.display = "none";
```

Comment:
```suggestion
  document.getElementById("submit_" + selector_id).style.display = "none";
```
selector_id is already there, it doesn't need to get it from the selector itself, and this could prevent a bug if the id attribute goes away, or selector becomes a different type at somepoint, etc. Best to not add more constraints than necessary on the "selector" interface.